src/govuk/components/button/button.yaml

```
@@ -43,6 +43,10 @@ params:
   type: boolean
   required: false
   description: Prevent accidental double clicks on submit buttons from submitting forms multiple times
+- name: isStartButton
+  type: boolean
+  required: false
+  description: If set to true this button will have an arrow icon added to it. Use for the main call to action on your service's start page.
```

Generally with a few exceptions we try to avoid explaining how to set options and what the internal implementation results from that.

I think we could make this simpler and just keep the latter bit of the sentence, something like:
```suggestion
  description: Use for the main call to action on your service's start page.
```

What do you think?

---

@nickcolley yes - that's better 👍